aerofiles/flarmcfg/writer.py

```diff
@@ -210,3 +220,39 @@ def write_waypoints(self, points):
                 raise ValueError('Invalid number of task point tuple items')
 
             self.write_waypoint(*args)
+
+    def write_field(self, key, value):
+        if key == "pilot":
+            self.write_pilot(value)
+        elif key == "copilot":
+            self.write_copilot(value)
+        elif key == "competition_class":
+            self.write_competition_class(value)
+        elif key == "glider_type":
+            self.write_glider_type(value)
+        elif key == "glider_id":
+            self.write_glider_id(value)
+        elif key == "competition_id":
+            self.write_competition_id(value)
+        elif key == "logger_interval":
+            self.write_logger_interval(value)
+        elif key == "task_name":
+            self.write_task_declaration(value)
+        elif key == "waypoints":
+            self.write_waypoints(value)
+        else:
+            raise ValueError("There is no functions to write %s" % key)
+
+    def write_declared_task(self, task_dict):
```

Comment:
ok i now get your point. the idea was that this dict comes from a different file type (for instance a cup file or tsk file).